src/vs/workbench/contrib/terminal/browser/media/terminal.css

```diff
@@ -248,11 +248,6 @@
 	top: 0;
 }
 
-.monaco-workbench .xterm .xterm-helper-textarea:focus {
-	/* Override the general vscode style applies `opacity:1!important` to textareas */
-	opacity: 0 !important;
-}
```

Comment:
I think we need this?

Reply:
Moved that to xterm.css

Reply:
I moved it up by the other textarea blocks